extensions/2.0/Khronos/KHR_gaussian_splatting/README.md

```diff
@@ -0,0 +1,389 @@
+  - [Extension Attributes](#extension-attributes)
+  - [Transforming Data](#transforming-gaussian-splat-data-for-gltf)
+- [Limitations](#limitations)
+- [Implementation](#implementation)
+- [Meshopt Compression](#meshopt-compression)
+- [Schema](#schema)
+- [Known Implementations](#known-implementations)
+- [Resources](#resources)
+
+## Overview
+
+Currently, PLY files serve as the de facto standard through their sheer simplicity and ubiquity. This extension aims to bring structure and conformity to the Gaussian Splat space while utilizing glTF to its fullest extent. Gaussian splats are essentially a superset of a traditional point cloud, so we approached the extension with this mindset. If the point primitive contains the extension the renderer can know to render the point primitive as Gaussian splats instead of a points. Gaussian splats are defined by a position, opacity, rotation, scale, and spherical harmonics (both diffuse and specular) . We store these values as attributes on primitives.
+
+This approach allows for an easy fallback in the event the glTF is loaded within a renderer that doesn't support Gaussian splats. In this scenario, the glTF file will render as a sparse point cloud to the user. It also allows for easy integration with existing compression like meshopt.
+
+## Adding Gaussian Splats to Primitives
+
+As Gaussian splats are defined by a position color, rotation and scale, we both map to existing attributes and define new ones. These are attached to a primitive  by defining the `extensions.KHR_gaussian_splatting` property on that mesh.
+
+### Splat Data Mapping
+
+To define a Gaussian splat, it must contain the following attributes:
+
+| Splat Data | glTF Attribute | Accessor Type | Component Type |
+| --- | --- | --- | --- |
+| Position | POSITION | VEC3 | float |
+| Color (Spherical Harmonic 0 (Diffuse) and opacity) | COLOR_0 | VEC4 | unsigned byte normalized |
+| Rotation | _ROTATION | VEC4 | float |
+| Scale | _SCALE | VEC3 | float |
+
+Standard PLY splat formats have opacity and color separate, however they are combined into the COLOR_0 attribute here.
+
+This implementation only contains the channel 0 spherical harmonic for diffuse color. Spherical Harmonic channels 1 through 15, which map the splat specular, are currently unused by the extension.
+
+Utilizing the standard position and color attributes allows us to easily fall back in situations where `extensions.KHR_gaussian_splatting` isn't available. Or simply for alternative rendering.
+
+### Extension attributes
+
+`extensions.KHR_gaussian_splatting` may contain the following values:
+
+| Attributes | Type | Description | Required |
+| --- | --- | --- | --- |
+| quantizedPositionScale | number | Scale value for dequantizing POSITION attribute values | No, default: `1.0` |
+
+```json
+{
+    "extensions": {
+        "KHR_gaussian_splatting": {
+            "quantizedPositionScale": 13.228187255859375
+        }
+    }
+}
+```
```

Comment:
Is `quantizedPositionScale` necessary? Usually with `KHR_mesh_quantization` the quantization scale is stored in the node transform.